staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -176,7 +176,7 @@ func TestCustomResourceValidation(t *testing.T) {
 	}
 	defer close(stopCh)
 
-	// enable alpha feature CustomResourceValidation
+	// enable feature CustomResourceValidation
 	err = utilfeature.DefaultFeatureGate.Set("CustomResourceValidation=true")
```

Comment:
can set block be dropped because it is enabled by default?

Reply:
yes. :+1:

Reply:
I dropped the various feature gate set blocks and amended the commit. Thanks.